Makefile

```diff
@@ -97,6 +97,6 @@ upgrade: ## update the pip requirements files to use the latest releases satisfy
 	# Post process all of the files generated above to work around open pip-tools issues
 	scripts/post-pip-compile.sh $(REQ_FILES:=.txt)
 	# Let tox control the Django version & django-oauth-toolkit version for tests
```

Comment:
just a minor suggestion, update the comment to remove django-oauth-toolkit since now we're only controlling Django's versions from here